geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -1,4 +1,61 @@
 {% load i18n corpus_extras %}
+<section id="transcription-panel">
+    {# NOTE: inputs must be kept as SIBLINGS of panel-container for CSS selection/controls #}
+    {# images displayed by default; disable if no images are available #}
+    <input type="checkbox" id="images-on"  {% if document.has_image %}checked="true" {% else %} disabled="true"{% endif %}/><label for="images-on">show images</label>
+    {# transcription displayed by default; disable if no content is available #}
+    <input type="checkbox" id="transcription-on" {% if document.digital_editions %}checked="true" {% else %} disabled="true"{% endif %}><label for="transcription-on">show transcription</label>
+    {# translation disabled for now since we don't handle them in the system yet #}
+    <input type="checkbox" id="translation-on" disabled="true"/><label for="translation-on">show translation</label>
+    <div class="panel-container">
+        {% comment %}
+NOTE: if we have more transcription pages than images,
+they won't show in current implementation.
+May want to gather them in the model... (but transcription content format will change)
+Also doesn't currently account for fragments without images.
+        {% endcomment %}
+        {# if we have images, loop based on them #}
+        {% if document.iiif_urls %}
+            {% for image_info in document.iiif_images %}
+                <div class="img">
+                    <h3>{{ image_info.shelfmark }} {{ image_info.label }}</h3>
+                    <img src="{{ image_info.image|iiif_image:"size:width=500" }}" alt="{{ image_info.label }}" title="{{ image_info.label }}" loading="lazy"
```

Comment:
I want OSD to be opt-in for efficiency and to avoid scroll traps. It should be enabled by clicking on the zoom controls in Gissoo's design. That work will be tracked under #891